Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
+            m 3eua "Not only is it beautiful, but it also helps humanity out."
+            m 4esc "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
+            m 3esd "Most of all, autotrophs such as plants and algae produce food and oxygen; without them, most life on Earth wouldn't exist."
```

Comment:
![1](https://user-images.githubusercontent.com/45746485/93428517-e7d29100-f8bf-11ea-94dd-a90dbc04a585.png)

Continuing in the same vein as before, she's more of a happy explainer. :)

Reply:
This sentence takes on more of a serious note though since autotrophs are literally so important that without them a majority of food webs would die off (including ours), and oxygen production on the planet would become virtually nonexistent. I prefer a more serious expression here, since while Monika can be happy in her explanations she can get serious too. I could do 3esc, but I think it'd be a bit of a jump to go from open smile to frown (I dunno though, c seems completely fine otherwise).

Reply:
I don't think this sentence takes on a serious tone at all. She's explaining life giving, beautiful parts of nature that are essential, I feel like she would be quite excited here. Like a kid with a train toy set explaining why these parts of the train keep the wheels spinning and without them the train wouldn't work. There's no dark tone about the train not working, the focus is on the thing that makes the other thing work. I hope I'm explaining my analogy correctly.

Reply:
It was a solid analogy. Fair enough, I'll change it.